modules/BookClass.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] Kindly remove this line and fix the linter errors because linter makes you write good code which follows best practices.